setup.py

```diff
@@ -13,8 +13,10 @@
     long_description=open('README.rst').read(),
     url='https://github.com/mozilla/FoxPuppet',
     license='MPL2',
-    packages=['foxpuppet'],
+    packages=[
+        'foxpuppet',
+        'foxpuppet.browser'],
```

Comment:
Ok, and the workaround to create the wheel from the sdist package is undesirable? If that is the case we could introduce find_packages later when the issue has been fully fixed.